pkg/sqlmigrations/migrations.go

```diff
@@ -105,60 +104,43 @@ var backwardCompatibleMigrations = []migrationDescriptor{
 		workFn: disableNetTrace,
 	},
 	{
-		// Introduced in v2.0.
-		// TODO(benesch): bake this migration into v2.1.
-		name:             "create system.table_statistics table",
-		workFn:           createTableStatisticsTable,
-		newDescriptorIDs: []sqlbase.ID{keys.TableStatisticsTableID},
+		// Introduced in v2.0. Baked into v2.1.
+		name: "create system.table_statistics table",
 	},
 	{
-		// Introduced in v2.0.
-		// TODO(benesch): bake this migration into v2.1.
+		// Introduced in v2.0. Permanent migration.
 		name:   "add root user",
 		workFn: addRootUser,
 	},
 	{
-		// Introduced in v2.0.
-		// TODO(benesch): bake this migration into v2.1.
-		name:             "create system.locations table",
-		workFn:           createLocationsTable,
-		newDescriptorIDs: []sqlbase.ID{keys.LocationsTableID},
+		// Introduced in v2.0. Baked into v2.1.
+		name: "create system.locations table",
 	},
 	{
-		// Introduced in v2.0.
-		// TODO(benesch): bake this migration into v2.1.
-		name:   "add default .meta and .liveness zone configs",
-		workFn: addDefaultMetaAndLivenessZoneConfigs,
+		// Introduced in v2.0. Baked into v2.1.
+		name: "add default .meta and .liveness zone configs",
 	},
 	{
-		// Introduced in v2.0.
-		// TODO(benesch): bake this migration into v2.1.
-		name:             "create system.role_members table",
-		workFn:           createRoleMembersTable,
-		newDescriptorIDs: []sqlbase.ID{keys.RoleMembersTableID},
+		// Introduced in v2.0. Baked into v2.1.
+		name: "create system.role_members table",
 	},
 	{
-		// Introduced in v2.0.
-		// TODO(benesch): bake this migration into v2.1.
-		name:         "add system.users isRole column and create admin role",
-		workFn:       addRoles,
-		doesBackfill: true,
+		// Introduced in v2.0. Permanent migration.
+		name:   "add system.users isRole column and create admin role",
+		workFn: addAdminRole,
```

Comment:
`doesBackfill` is dropped since the schema part of this migration is now baked in. I've left the `skip backfill` testing knob, mostly because it was so incredibly tedious to fix all this the first time.